configuration/mongodb.go

```diff
@@ -0,0 +1,40 @@
+package configuration
+
+import (
+	"context"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	userCollection *mongo.Collection
+	ctx            = context.TODO()
```

Comment:
in Golang, it is a common and idiomatic practice to pass the context explicitly as an argument rather than using global variables. This approach aligns with the principles of simplicity, readability, and flexibility that are emphasized in the Go language.